### ISSUE TITLE
v2/codestructure does not detect NUnit's Theories

### ISSUE THREAD

**Issue by Melandel** [OmniSharp/omnisharp-roslyn#2379]
Hello,

Here's a [repository I made for reproducing](https://github.com/Melandel/demo.omnisharpvim.bug.nunitTheory) the following observation: 

* `v2/codestructure` detects a test on the following code:
```cs
using NUnit.Framework;

namespace demo.omnisharpvim.bug.nunitTheory
{
	public enum Color { Blue, White, Red }
	public class Tests
	{

// Logs show the following properties for the method Test1 below:
//  "Properties": {
//    "testMethodName": "demo.omnisharpvim.bug.nunitTheory.Tests.Test1",
//    "accessibility": "public",
//     "testFramework": "nunit",
//     "static": false
//  }
		[TestCase(Color.Blue)]
		[TestCase(Color.White)]
		[TestCase(Color.Red)]
		public void Test1(Color color)
		{
			Assert.That(new[] { "Blue", "White", "Red" }, Does.Contain(color.ToString()));
		}
	}
}
```
* However, `v2/codestructure` does not detect the same test when written under the following form:
```cs
using NUnit.Framework;

namespace demo.omnisharpvim.bug.nunitTheory
{
	public enum Color { Blue, White, Red }
	public class Tests
	{

// Logs show the following properties for the method Test1 below:
//  "Properties": {
//    "accessibility": "public",
//     "static": false
//  }
		[Theory]
		public void Test1(Color color)
		{
			Assert.That(new[] { "Blue", "White", "Red" }, Does.Contain(color.ToString()));
		}
	}
}
```

Is this behaviour expected?
